buku

```diff
@@ -725,9 +755,8 @@ class BukuDb:
             for index in range(low, high + 1):
                 self.compactdb(index)
         elif index == 0:  # Remove the table
-            resp = input('ALL bookmarks will be removed. Enter \x1b[1my\x1b[21m to confirm: ')
-            if resp != 'y':
-                print('No bookmarks deleted')
+            input_func = input_func or self.prompt_delete_all_bookmarks
+            if self.delete_all_bookmarks(input_func) == None:
```

Comment:
Why are we calling delete_all_bookmarks() twice? Why not just check for input_func?

Also, this leads to an issue:

```
$ buku -d
ALL bookmarks will be removed. Enter y to confirm: y
Traceback (most recent call last):
  File "/usr/local/bin/buku", line 1763, in <module>
    bdb.delete_bookmark(0)
  File "/usr/local/bin/buku", line 762, in delete_bookmark
    self.delete_all_bookmarks()
TypeError: delete_all_bookmarks() missing 1 required positional argument: 'input_func'
```